mysql-test/suite/rocksdb/t/skip_validate_tmp_table.test

```diff
@@ -0,0 +1,26 @@
+--source include/have_rocksdb.inc
+--source include/have_debug.inc
+
+# Write file to make mysql-test-run.pl expect the "crash", but don't restart the
+# server until it is told to
+--source include/expect_crash.inc
```

Comment:
I don't see any crash-causing statement between here and restart_mysqld.inc ?

Reply:
Ahh, yeah, right. This is a mistake in updating the test when I imported it.